tests/acceptance/features/apiAntivirus/antivirus.feature

```diff
@@ -293,4 +293,26 @@ Feature: antivirus
       | dav-path-version |
       | old              |
       | new              |
-      | spaces           |
\ No newline at end of file
+      | spaces           |
+
+  @issue-enterprise-5709
+  Scenario Outline: try to create a version of file by uploading virus content
```

Comment:
```suggestion
  Scenario Outline: create a version of file by uploading virus content
```

Reply:
IMO, the scenario tries to create a version (version creation fails). So `try to ...` best suites this scenario